src/vl-map-modify-action.js

```diff
@@ -0,0 +1,22 @@
+import {VlMapLayerAction} from '/src/vl-map-layer-action.js';
+import {VlModifyAction} from '/node_modules/vl-mapactions/dist/vl-mapactions.js';
+
+/**
+ * VlMapModifyAction
+ * @class
+ * @classdesc De kaart aanpas actie component.
+ *
+ * @extends VlMapLayerAction
+ *
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-map/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-map/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-map-modify-actions.html|Demo}
+ */
+export class VlMapModifyAction extends VlMapLayerAction {
+  _createAction(layer) {
+    const options = {
+      snapping: true,
+    };
```

Comment:
dit is dan niet het "snappen op punten van een bepaalde kaartlaag" dat we bij oppervlaktewater gebruiken maar een ander soort snapping?

Reply:
ik had het concept van snapping gevonden in de voorbeelden van OL en dan gaan kijken of we dit zelf ook al ondersteunen. Nu dit leek me hier hetzelfde concept. De laag waarop hij dit toepast is by default de layer die we meegeven aan de modify constructor. Als er nog een andere concept van snapping, hier wou ik alvast die van OL gebruiken.